public/app/plugins/datasource/cloudwatch/components/MetricStatEditor/MetricStatEditor.tsx

```diff
@@ -109,7 +109,8 @@ export function MetricStatEditor({
             tooltip="Only show metrics that exactly match all defined dimension names."
           >
             <Switch
-              checked={!!query.matchExact}
+              id="cloudwatch-match-exact"
```

Comment:
is this id necessary ?

Reply:
yeah otherwise react testing library selector won't work. see https://github.com/grafana/grafana/blob/cloudwatch/42618-match-exact-default-value/public/app/plugins/datasource/cloudwatch/components/ui/EditorField.tsx#L20